### ISSUE TITLE
USER ISSUE: Server Crash but just for me ?

### ISSUE THREAD

**Issue by Hybris95** [StrangeLoopGames/EcoIssues#24206]
\
**User Id: (Not logged user)**
**Version:** 0.10.0.0 beta release-532

**Steps to Reproduce:**
Big server lagging all day long, and sometimes crash happen

**Expected behavior:**
No lags, no crash

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: IndexOutOfRangeException
Message:Index was outside the bounds of the array.
Source:Eco.Gameplay

System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Eco.Gameplay.Settlements.EcopediaSettlementConfigData.GetEcopediaData(Player player, EcopediaPage page)
   at </size>...

Check log file to see full message.
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by Hybris95** [comment]
[EcoLog.txt](https://github.com/StrangeLoopGames/EcoIssues/files/13678470/EcoLog.txt)

**Comment by Hybris95** [comment]
[EcoLog_withException.txt](https://github.com/StrangeLoopGames/EcoIssues/files/13683771/EcoLog_withException.txt)

**Comment by Hybris95** [comment]
The server holds from 130 to 200 people.
When population is below 50 usually everything is fine.

**Comment by D3nnis3n** [comment]
Fixed in 10.0.1
